### ISSUE TITLE
ti-community-bot's SIG committer promotion check is too strict

### ISSUE THREAD

**Issue by kennytm** [pingcap/community#382]
Currently, ti-community-bot requires the following roles to /lgtm on a SIG's membership.json file:

1. modifying "activeContributor" requires one /lgtm from reviewers (or above)
2. modifying "reviewers" requires one /lgtm from committers (or above)
3. **modifying "committers" requires two /lgtm from "maintainers"**
4. modifying "coLeaders" and "techLeaders" require two /lgtm from "maintainers"

The permission checks for 1, 2 and 4 are mostly consistent with existing SIG promotion rules, but for 3 it is too strict for most SIGs. 

<details><summary>Let's check the existing roles-and-organization-management.md of all SIGs:</summary>

* SIG-Community-Infra:

    https://github.com/pingcap/community/blob/3dc8e3d43db39b59bec36fa2c9033ed5740b72fd/special-interest-groups/sig-community-infra/roles-and-organization-management.md#L39

* SIG-Diagnosis:

    https://github.com/pingcap/community/blob/3dc8e3d43db39b59bec36fa2c9033ed5740b72fd/special-interest-groups/sig-diagnosis/roles-and-organization-management.md#L53-L54

* SIG-Docs:

    https://github.com/pingcap/community/blob/3dc8e3d43db39b59bec36fa2c9033ed5740b72fd/special-interest-groups/sig-docs/roles-and-organization-management.md#L45

* SIG-Exec:

    https://github.com/pingcap/community/blob/3dc8e3d43db39b59bec36fa2c9033ed5740b72fd/special-interest-groups/sig-exec/roles-and-organization-management.md#L35

* SIG-K8s:

    https://github.com/pingcap/community/blob/3dc8e3d43db39b59bec36fa2c9033ed5740b72fd/special-interest-groups/sig-k8s/roles-and-organization-management.md#L38

* SIG-Migrate:

    https://github.com/pingcap/community/blob/3dc8e3d43db39b59bec36fa2c9033ed5740b72fd/special-interest-groups/sig-migrate/roles-and-organization-management.md#L44

* SIG-Planner:

    (none 🤔)

* SIG-SQL-Infra:

    https://github.com/pingcap/community/blob/3dc8e3d43db39b59bec36fa2c9033ed5740b72fd/special-interest-groups/sig-sql-infra/roles-and-organization-management.md#L35

* SIG-TiUp

    https://github.com/pingcap/community/blob/3dc8e3d43db39b59bec36fa2c9033ed5740b72fd/special-interest-groups/sig-tiup/roles-and-organization-management.md#L39

* SIG-Web:

    https://github.com/pingcap/community/blob/3dc8e3d43db39b59bec36fa2c9033ed5740b72fd/special-interest-groups/sig-web/roles-and-organization-management.md#L39

</details>

Basically, other than SIG-Diagnosis, all SIG only require approval from 2 existing committers or maintainers. Restricting /lgtm to maintainers is inconsistent with the existing rules. So either the bot should be changed to follow the common denominator of `roles-and-organization-management.md`, or all `roles-and-organization-management.md` should be changed to align with the bot.

**Comment by hi-rustin** [comment]
I fixed some mistakes you made in the permission description. Now it's aligned with the robot.

**Comment by hi-rustin** [comment]
In fact, I want to redesign the whole promotion process, I think the current way of merging is too inefficient. I will write a document to redesign the process, and maybe we can change the permission to add committers before that.

@winkyao @sykp241095  What do you think?

**Comment by sykp241095** [comment]
+1, the current procedure is tooooo complex.

**Comment by hi-rustin** [comment]
/assign

I will submit a new process design to optimize the process.

**Comment by hi-rustin** [comment]
> I will submit a new process design to optimize the process.

See: https://docs.google.com/document/d/1OvlpU00W_LpTL09zB3gvDyQzHULz27kmNKH2O0seWCI/edit?usp=sharing

**Comment by hi-rustin** [comment]
/close 

By https://github.com/pingcap/community/pull/402
